backend/database.js

```diff
@@ -173,9 +174,8 @@ export default async () => {
   // TODO: Update this to only run when persistence is disabled when `chel deploy` can target SQLite.
   if (persistence !== 'fs' || options.fs.dirname !== dbRootPath) {
     // Remember to keep these values up-to-date.
-    const HASH_LENGTH = 52
-    const CONTRACT_MANIFEST_MAGIC = '{"head":"{\\"manifestVersion\\"'
-    const CONTRACT_SOURCE_MAGIC = '"use strict";'
+    const HASH_LENGTH = 56
```

Comment:
Interesting that the hash length changed... are you sure it's 56 then for all of the ones we're checking for here?

Reply:
Yes, you can also verify this by looking at the hashes generated. The reason the length changed is that the content id is a varint, and the new values are longer than the previous one (0x00 I believe, which is encoded as a single octet)